build-and-test.ps1

```diff
@@ -0,0 +1,81 @@
+[cmdletbinding()]
+param(
+    [string]$VersionFilter,
+    [string]$OSFilter,
+    [switch]$IsDryRun
+)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+
+$buildFilter = "*"
+if (-not [string]::IsNullOrEmpty($VersionFilter))
```

Comment:
What do you think about defaulting VersionFilter and OSFilter to '*' if not specified?  Then this logic could be simplified to - `buildFilter = "{VersionFilter}-{OSFilter}/*"`

Reply:
Yes, this works. Thanks!